dali/test/python/test_RN_external_source_parallel.py

```diff
@@ -0,0 +1,291 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Can you also add a test for correctness - if we get then same results from the multiprocessing case as for the single process one?

Reply:
Sure!

Reply:
Added various correctness tests in test_external_source_parallel